### PR TITLE
outline of milkjmonster lesson

### PR DESCRIPTION
**Pull request by pelikhan**
DO NOT MERGE.

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
         * [Duct tape wallet](/projects/wallet)
         * [Watch](/projects/watch)
         * [Inchworm](/projects/inchworm)
+        * [Milk moster](/projects/milk-monster)
```
Comment (pelikhan) [inline docs/SUMMARY.md]: Should it go above inchworm?

File: docs/projects.md [hunk 2]
```diff
@@ -69,6 +69,10 @@ Fun games to build with your @boardname@.
   "name": "Inchworm",
   "url":"/projects/inchworm",
   "imageUrl":"/static/mb/projects/inchworm.jpg"
+}, {
+  "name": "Milk monster",
+  "url":"/projects/milk-monster",
+  "imageUrl":"/static/mb/projects/milk-monster.jpg"
```
Comment (pelikhan) [inline docs/projects.md]: image missing, should be 303x200

File: docs/projects/milk-monster.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+https://youtu.be/sPGxN4UIpgk?t=10s
+
+## Duration
+
+3 Activities, approx 30-45 min each based on familiarity with the coding concepts
+
+## Materials
+
+Recycled
+* 1 Milk Carton (recycle!)
```
Comment (pelikhan) [inline docs/projects/milk-monster.md]: specify that it needs to have a screwable top

File: docs/projects/milk-monster.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+https://youtu.be/sPGxN4UIpgk?t=10s
+
+## Duration
+
+3 Activities, approx 30-45 min each based on familiarity with the coding concepts
+
+## Materials
+
+Recycled
```
Comment (pelikhan) [inline docs/projects/milk-monster.md]: ``####`` menu level

File: docs/projects/milk-monster.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* 3 Crocodile clips
+* 1 micro servo 9g SG90
+* 1 servo clip designed for Milky Monster [download on thingiverse](http://www.thingiverse.com/thing:2185971) (optional). Teachers may prefer to download the classroom pack (16 servo clips).
+
+Tools
+* Small knive and  Scissors that can cut cardboard
+* Tape (masking, duct tape, and/or packing tape)
+* Glue gun
+
+![Materials](/static/mb/projects/milk-monster/materials.jpg)
```
Comment (pelikhan) [inline docs/projects/milk-monster.md]: remove text from picture (can't localize)

File: docs/projects/milk-monster/make.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+## Step 1: download the template
+
+[Download the Milky Monster template](https://msenmediastorage.blob.core.windows.net/resources/f12f968b-bb68-41d1-a77f-700dab181042.pdf) and cutout the template.
+
+
+![](/static/mb/projects/milk-monster/cut template.jpg)
+https://youtu.be/ipsZuFNgTHM
+
+## Step 1: cut the template
```
Comment (pelikhan) [inline docs/projects/milk-monster/make.md]: duplicate step

File: docs/projects/milk-monster/make.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+* 1 servo clip designed for Milky Monster [download on thingiverse](http://www.thingiverse.com/thing:2185971) (optional). Teachers may prefer to download the classroom pack (16 servo clips).
+
+Tools
+* Small knive and  Scissors that can cut cardboard
+* Tape (masking, duct tape, and/or packing tape)
+* Glue gun
+
+## Step 1: download the template
+
+[Download the Milky Monster template](https://msenmediastorage.blob.core.windows.net/resources/f12f968b-bb68-41d1-a77f-700dab181042.pdf) and cutout the template.
```
Comment (pelikhan) [inline docs/projects/milk-monster/make.md]: store pdf under /static/mb/projects/milk-monster/...

File: docs/projects/milk-monster/make.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+https://youtu.be/ipsZuFNgTHM
+
+## Step 1: cut the template
+
+[Download the Milky Monster template](https://msenmediastorage.blob.core.windows.net/resources/f12f968b-bb68-41d1-a77f-700dab181042.pdf) and cutout the template. 
+
+![](/static/mb/projects/milk-monster/cut template.jpg)
+https://youtu.be/ipsZuFNgTHM
+
+## Step 2: glue template on milk carton
```
Comment (pelikhan) [inline docs/projects/milk-monster/make.md]: unclear what to do here

File: docs/projects/milk-monster/make.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+Create space to route the cables nicely on back of the milk-monster.
+
+https://youtu.be/BoIRCk769MM
+
+## Step 9: attaching clip to servo
+
+Use a philips screw driver to mount the servo to the 3D-printed servo clip.
+If you don't have the 3D-printed clips available, [download on thingiverse](http://www.thingiverse.com/thing:2185971) , or fix the servo using tyraps or a glue gun.
+
+![](/static/mb/projects/milk-monster/clip to servo.jpg)
```
Comment (pelikhan) [inline docs/projects/milk-monster/make.md]: broken image

File: docs/projects/milk-monster/make.md (hunk 7 above)
Comment (peterheldens) [inline docs/projects/milk-monster/make.md]: tx, done

File: docs/projects/milk-monster/make.md (hunk 8 above)
Comment (peterheldens) [inline docs/projects/milk-monster/make.md]: video was missing; now added.

File: docs/projects/milk-monster/make.md (hunk 9 above)
Comment (peterheldens) [inline docs/projects/milk-monster/make.md]: fixed

File: docs/projects/milk-monster/make.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+* 1 servo clip designed for Milky Monster [download on thingiverse](http://www.thingiverse.com/thing:2185971) (optional). Teachers may prefer to download the classroom pack (16 servo clips).
+
+Tools
+* Small knive and  Scissors that can cut cardboard
+* Tape (masking, duct tape, and/or packing tape)
+* Glue gun
+
+## Step 1: download and cut the template
+
+[Download the Milky Monster template]![](/static/mb/projects/milk-monster/cut template.jpg) and cutout the template.
```
Comment (pelikhan) [inline docs/projects/milk-monster/make.md]: missing image

File: docs/projects/milk-monster/connect.md [hunk 11]
```diff
@@ -25,4 +25,6 @@ input.onButtonPressed(Button.A, () => {
 
 ```package
 radio
-```
\ No newline at end of file
+```
+## about the author
```
Comment (pelikhan) [inline docs/projects/milk-monster/connect.md]: should go in the "landing page" of the project

File: docs/projects/milk-monster/make.md (hunk 10 above)
Comment (peterheldens) [inline docs/projects/milk-monster/make.md]: fixed

**Pull request merged**